checks.d/mysql.py

```diff
@@ -535,17 +535,17 @@ def _collect_metrics(self, host, db, tags, options, queries):
             # slaves will only be collected iff user has PROCESS privileges.
             slaves = self._collect_scalar('Slaves_connected', results)
 
-            if slave_running is not None:
+            if slave_running and not slaves:  # slave
```

Comment:
It think `slaves` can only be a ` float`, wouldn't it be better to do a `slaves == 0.` ?